packages/common-all/src/uuid.ts

```diff
@@ -1,14 +1,15 @@
 import { customAlphabet as nanoid } from "nanoid";
 import { customAlphabet as nanoidInsecure } from "nanoid/non-secure";
-import { alphanumeric } from "nanoid-dictionary";
 
 /** Using this length, according to [nanoid collision calculator](https://zelark.github.io/nano-id-cc/),
  * generating 1000 IDs per hour, it would take around 919 years to have 1 percent chance of a single collision.
  * This is okay for the "insecure" generator, which is used in limited cases where collisions are less likely.
  */
 const SHORT_ID_LENGTH = 12;
 /** Default length for nanoids. */
-const LONG_ID_LENGTH = 21;
+const LONG_ID_LENGTH = 32;
```

Comment:
Using the collision calculator linked in the comment, bumping the ID length from 21 to 23 with the dictionary change in this PR is enough to preserve the collision resistence. Shorter IDs mean shorter links, so I'd suggesst lowering this to 23.

Reply:
updated!